pelican/utils.py

```diff
@@ -36,6 +36,18 @@
 logger = logging.getLogger(__name__)
 
 
+def sanitised_join(base_directory, *parts):
+    joined = os.path.abspath(os.path.join(base_directory, *parts))
+    if not joined.startswith(os.path.abspath(base_directory)):
+        raise RuntimeError(
+            "attempt to break out of output directory to {}".format(
```

Comment:
Minor capitalization/tense nitpick: `Attempted to break out of output directory to {}`.